src/util/atoms.ts

```diff
@@ -2,12 +2,14 @@
 
 import { atom, createStore } from "jotai";
 import { AppConfig, UserSession, UserData } from "@stacks/connect";
-import { BITCOIND_URL } from "@/app/api/proxy/[...proxy]/route";
 
 import { StacksNetwork, STACKS_TESTNET } from "@stacks/network";
 
 export const store = createStore();
 
+const BITCOIND_URL =
+  process.env.NEXT_PUBLIC_BITCOIND_URL || "http://localhost:18443";
+
```

Comment:
nit: duplicated code can be easily removed away by keeping it in a common file and importing it

Reply:
Fixed in [a9695fd](https://github.com/stacks-network/sbtc-bridge/pull/17/commits/a9695fd8bb6c7eed9142330d3c98d2a1cbbd9894)